### PR TITLE
update readme

### LINKED ISSUE #37: Update Readme with instructions
Instructions about contributing, branching, etc.

### PR DESCRIPTION
**Pull request by elia314**
I didn't know what image you wanted to replace in Peek at Curiosity, thoughts? I also didn't know how to update the pull request #53 with this edit, so I created this pull request.

Closes #37

@alejandronanez @mubaris

### CODE REVIEW

**Comment by maiquynhtruong** [comment]
I uploaded an image and referred that in a comment in #37. Here is the link again http://i.imgsafe.org/c45b215899.jpg. You can run this project on your computer and take a screenshot.

**Comment by elia314** [comment]
Thanks @maiquynhtruong, I updated the image. Please let me know if there is any more updates I can help with the readme.

File: README.md [hunk 1]
```diff
@@ -7,13 +7,14 @@
 [![Build Status](https://travis-ci.org/curiositylab/curiosity.svg?branch=master)](https://travis-ci.org/curiositylab/curiosity)
 [![Say Thanks](https://img.shields.io/badge/Say%20Thanks!-%F0%9F%A6%89-1EAEDB.svg)](https://saythanks.io/to/mubaris)
 
+## Features
 Find amazing github projects as a feed. :zap: :zap:
-
+Curiosity is a web app.
```
Comment (mubaris) [inline README.md]: You don't have to mention it's web app.

Comment (mubaris) [inline README.md]: You need to add details about branching and contributing.

Comment (mubaris) [inline README.md]: Do not create new readme. Edit the existing one.

**Comment by mubaris** [comment]
If you have issues about git. Follow this [tutorial](https://try.github.io/levels/1/challenges/1)

**Comment by elia314** [comment]
Perhaps I'm not understanding how to use this web app. How do you use it without the Chrome extension? I'm not sure how to go about branching and contributing. I did update minor edits in #55 though.

**Comment by elia314** [comment]
I did go through git tutorial, thank you that was helpful. If you mean git hub branching and contributing related to this repo, then I can write some instructions for that if that's what you mean @mubaris .

**Comment by mubaris** [comment]
I want you to write about all the things I mentioned in #37.

You can access webapp here - [https://curiositylab.github.io/curiosity/](https://curiositylab.github.io/curiosity/)

**Comment by elia314** [comment]
I made updates per #37, please let me know if those are ok or if you request edits @mubaris.

File: README.md [hunk 2]
```diff
@@ -27,16 +35,24 @@ Curiosity Chrome extension is available now. Which is made by [Aswanth](https://
...
 
 * [SweetAlert2](https://limonte.github.io/sweetalert2/)
 * [infinite-scroll](https://github.com/alexblack/infinite-scroll)
 * [Axios](https://github.com/mzabriskie/axios)
 
+## Labels
+* `up-for-grabs` labels are the issues need to solved. 
+* `discussion` label is for discussions of tasks. 
+* `enhancement` are features to make project better.
+* `testing` are features that aren't implemented yet but are being tested.
```
Comment (mubaris) [inline README.md]: Testing is not for the future features. It's for unit testing, code testing and code coverage.

File: README.md [hunk 3]
```diff
@@ -27,16 +35,24 @@ Curiosity Chrome extension is available now. Which is made by [Aswanth](https://
...
+* `difficulty` and whatever degree it is assesses the skill needed to help with that task
+* `code style` are features/tasks to integrate code better in the project
+
+## Contributing/notes
+For branching/contributing to this repo, we have 3 branches. `gh-pages` contains only front-end and it's hosted on Github Pages. `v2` is the future of Curiosity. We are building a backend for this. The code for that is contained in `v2`. master is main branch. `v2` will be merged to `master` once the process of building backend is done. Please feel free to contribute!
```
Comment (mubaris) [inline README.md]: Change the first sentence into "We maintain 3 branches"

**Comment by elia314** [comment]
Made those updates @mubaris . Please review.

As for adding myself as a contributor using yarn all-contributors:add, is that in terminal or in the actual readme file?

**Comment by elia314** [comment]
Awesome, thanks @mubaris .

If I can be added as a contributor, how do I go about doing that? Using yarn all-contributors:add, is that in terminal or writing it in the readme file?

**Comment by mubaris** [comment]
You can run `yarn all-contributors add` and enter your username, select Documentation in terminal, then send a PR.

**Comment by elia314** [comment]
image

I tried that in my terminal, yarn or npm aren't found. Perhaps I need to install other packages before I can do this?

**Comment by mubaris** [comment]
Install both npm and yarn for macOS. Then `yarn install` press Enter -> `yarn all-contributors add` press Enter. Fill the details.

**Pull request merged**